rust/src/models/product.rs

```diff
@@ -49,16 +49,28 @@ pub enum FixedPackage {
     ByFullName {
         full_name: String,
         specifier: Specifier,
+        app_stream: Option<AppStream>,
```

Comment:
```suggestion
        module: Option<AppStream>,
```
Any objection to call this `module` or `app_stream_module`? I'm asking because "AppStream" is only the RPM repository, but we're comparing modules (which are part of the AppStream repo).